services/__tests__/location.spec.js

```diff
@@ -0,0 +1,82 @@
+import { jest } from '@jest/globals';
+
+import {
+  getCountriesProvider,
+  getRegionsProvider,
+  getSubRegionsProvider,
+} from 'services/country';
+
+import { countryConfig } from '../location';
+
+jest.mock('services/country', () => ({
+  getCountriesProvider: jest.fn(),
+  getRegionsProvider: jest.fn(),
+  getSubRegionsProvider: jest.fn(),
+}));
+
+describe('countryConfig', () => {
```

Comment:
Having these great tests should be very helpful if you decide to tackle my recommended refactor of `countryConfig` mentioned earlier in this review.

Reply:
It was very helpful! haha